src/js/media-container.js

```diff
@@ -245,7 +241,7 @@ function getBreakpoints(breakpoints, rect) {
  * @attr {boolean} userinactive
  *
  * @cssprop --media-background-color - `background-color` of container.
- * @cssprop --media-element-display - `display` of the slotted media element (default none for [audio] usage).
+ * @cssprop --audio-element-display - `display` of the slotted media element when the audio attribute is present.
```

Comment:
the default value of `none` is automatically added in the custom elements manifest